cwms-data-api/src/main/java/cwms/cda/data/dao/project/ProjectLockDao.java

```diff
@@ -147,12 +141,26 @@ private List<ProjectLock> catLocks(String projMask, String appMask, TimeZone tz,
                 ;
     }
 
-    public void releaseLock(String lockId) {
+    /**
+     * Releases a lock for a given office and lock ID.
+     *
+     * @param office  the office to be used for the session.
+     * @param lockId  the ID of the lock to release
+     */
+    public void releaseLock(String office, String lockId) {
         connection(dsl, c -> CWMS_PROJECT_PACKAGE.call_RELEASE_LOCK(
-                DSL.using(c, SQLDialect.ORACLE18C).configuration(),
```

Comment:
consider if it makes sense to suppress the line length warning here. Alternatively nesting the calls in a way that makes it faster to spot which items are an argument to what would be nice.

Reply:
I pulled the configuartion arg into its own variable and adjusted the indents